Makefile

```diff
@@ -117,7 +117,7 @@ ifeq ("$(OS)","windows")
 BINARIES=$(BUILDDIR)/tsh
 endif
 
-VERSRC = version.go gitref.go
+VERSRC = version.go gitref.go api/version.go
```

Comment:
let's add a make target to re-generate all of these easily:
```make
.PHONY: version
version: $(VERSRC)
```
and update the generated comment text to mention `make version`